guides/source/action_cable_overview.md

```diff
@@ -0,0 +1,659 @@
+if the user can be identified.
+
+#### Connection Setup
+```ruby
+# app/channels/application_cable/connection.rb
+module ApplicationCable
+  class Connection < ActionCable::Connection::Base
+    identified_by :current_user
+
+    def connect
+      self.current_user = find_verified_user
+    end
+
+    protected
+      def find_verified_user
+        if current_user = User.find_by(id: cookies.signed[:user_id])
+          current_user
+        else
+          reject_unauthorized_connection
+        end
+      end
+  end
+end
+```
+Here `identified_by` is a connection identifier that can be used to find the
+specific connection again or later.
+Note that anything marked as an identifier will automatically create a delegate
+by the same name on any channel instances created off the connection.
+
+This relies on the fact that you will already have handled authentication of the user,
+and that a successful authentication sets a signed cookie with the `user_id`. 
+This cookie is then automatically sent to the connection instance when a new connection
+is attempted, and you use that to set the `current_user`. By identifying the connection
+by this same current_user, you're also ensuring that you can later retrieve all open
+connections by a given user (and potentially disconnect them all if the user is deleted
+or deauthorized).
+
+### Channels
+A channel encapsulates a logical unit of work, similar to what a controller does in a
+regular MVC setup. 
+You should first define a parent `ApplicationCable::Channel` class in Ruby, which
+encapsulates shared logic between your channels. 
+
+#### Parent Channel Setup
+```ruby
+# app/channels/application_cable/channel.rb
+module ApplicationCable
+  class Channel < ActionCable::Channel::Base
+  end
+end
+```
+Then you would create child channel classes. For example, you could have a
+**ChatChannel** and an **AppearanceChannel**:
+
+```ruby
+# app/channels/application_cable/chat_channel.rb
+class ChatChannel < ApplicationCable::Channel
+end
+
+# app/channels/application_cable/appearance_channel.rb
+class AppearanceChannel < ApplicationCable::Channel
+end
+```
+
+ A consumer could then be subscribed to either or both of these channels.
+
+#### Subscriptions
+
+When a consumer is subscribed to a channel, they act as a subscriber;
+This connection is called a subscription.
+Incoming messages are then routed to these channel subscriptions based on
+an identifier sent by the cable consumer.
+
+```ruby
+# app/channels/application_cable/chat_channel.rb
+class ChatChannel < ApplicationCable::Channel
+  def subscribed
+  end
+end
+```
+
+## Client-Side Components
+### Connections
+Consumers require an instance of the connection on their side. This can be
+established using the following Javascript:
+
+#### Connect Consumer
+```coffeescript
+# app/assets/javascripts/cable.coffee
+#= require action_cable
+
+@App = {}
+App.cable = ActionCable.createConsumer("ws://cable.example.com")
```

Comment:
This will change once we go default on /cable. Then presumably we'll just have a call to `ActionCable.createConsumer`. But we can explain then use of the action_cable_meta_tag here to configure this.